Utilities/Utilities/_Utils/_FeatureFlags/CompositeFeatureFlagsProvider.swift

```diff
@@ -12,12 +12,6 @@ import Combine
 public class CompositeFeatureFlagsProvider: NSObject & FeatureFlagsProtocol {
     public var local: FeatureFlagsProtocol?
     public var remote: FeatureFlagsProtocol?
-    
-    public var newValuesAvailablePublisher: AnyPublisher<Void, Never> {
```

Comment:
no longer necessary since we are keeping session values constant for entire session after first access